github-repo/action.yaml

```diff
@@ -0,0 +1,55 @@
+name: "Mondoo Github Repository Action"
+description: "Scan Github repository for misconfigurations with Mondoo"
```

Comment:
```suggestion
description: "Scan Github repositories for misconfigurations with Mondoo"
```

Reply:
This actions is actually optimized for scanning one specific repository